lib/modules/editor/utils/editor_sheet.dart

```diff
@@ -85,15 +85,17 @@ Future<T?> showEditorSheet<T>({
               const SizedBox(width: 4),
             ],
           ),
-          floatingActionButton: FloatingActionButton(
-            onPressed: () {
-              if (form.currentState?.validate() ?? false) {
-                Navigator.pop(context, true);
-              }
-            },
-            child: const Icon(Icons.done_all_rounded),
-            tooltip: 'Save',
-          ),
+          floatingActionButton: onSave == null
+              ? null
+              : FloatingActionButton(
+                  onPressed: () {
+                    if (form.currentState?.validate() ?? false) {
+                      Navigator.pop(context, true);
+                    }
+                  },
+                  child: const Icon(Icons.done_all_rounded),
+                  tooltip: 'Save',
+                ),
           body: StatefulBuilder(
```

Comment:
One thing I realized about this editor sheet is that if you drag it down from its title bar, it doesn't show the "Discard changes" dialog

Reply:
Yes, unfortunately, it's a know issue since the beginning. I haven't figured it out yet.